icu4j/main/tests/core/src/com/ibm/icu/dev/test/timezone/TimeZoneRuleTest.java

```diff
@@ -1235,7 +1236,7 @@ public void TestVTimeZoneCoverage() {
 
         int[] offsets_vtzc = new int[2];
         VTimeZone vtzc = VTimeZone.create("PST");
-        vtzc.getOffsetFromLocal(Calendar.getInstance(vtzc).getTimeInMillis(), VTimeZone.LOCAL_STD, VTimeZone.LOCAL_STD, offsets_vtzc);
+        vtzc.getOffsetFromLocal(Calendar.getInstance(vtzc).getTimeInMillis(), LocalOption.STANDARD_FORMER, LocalOption.STANDARD_LATTER, offsets_vtzc);
         if (offsets_vtzc[0] > offsets_vtzc[1]) {
             errln("Error getOffsetFromLocal()");
         }
```

Comment:
could we extend the test to cover the testing of all 6 possible values LocalOption for VTimeZone here?

Reply:
As I mentioned in #1610, I don't want to rush it today. I suggest to work on it under a new ticket.

Reply:
could you file a ticket and add a TODO with that ticket number here at least. thanks

Reply:
https://unicode-org.atlassian.net/browse/ICU-21516